composer.json

```diff
@@ -24,12 +24,14 @@
     "automattic/vipwpcs": "^2.3",
     "dealerdirect/phpcodesniffer-composer-installer": "^0.7",
     "dg/bypass-finals": "1.1.3",
+    "php-stubs/wordpress-stubs": "^5.9",
```

Comment:
This is a transitive dependency of both `szepeviktor/phpstan-wordpress` and `sniccwp/php-scoper-wordpress-excludes`, so we shouldn't need to add it here explicitly